setup.cfg

```diff
@@ -0,0 +1,30 @@
+[metadata]
+name = pyflow
+version = attr: pyflow/version.__version__
+author = European Centre for Medium-Range Weather Forecasts (ECMWF)
+author_email = software.support@ecmwf.int
+license = Apache 2.0
+license_files = LICENSE
+description = Create pythonic ecFlow suites
+long_description = file: README.md
+long_description_content_type=text/markdown
+url = https://pyflow-workflow-generator.readthedocs.io/en/
+
+[options]
+packages = find_namespace:
+include_package_data = True
+install_requires =
+    jinja2
+    requests
```

Comment:
Maybe we could also pin the versions of the dependencies? i.e. `jinja2==2.*` or so

Reply:
And I was going to add `ecflow` from PyPI when I realized it's not deployed there. I went to ecmwf/ecflow to try to install it from a tag, when I remembered it's a bind to the C++ code. I guess it should be possible to upload a version for that too in PyPI?